Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="18" android:versionName="0.10.1-SNAPSHOT" package="com.microsoft.azure.mobile.xamarin.forms.puppet">
 	<uses-sdk android:minSdkVersion="15" />
-	<application android:label="MCFPuppet"></application>
+	<uses-permission android:name="android.permission.INTERNET" />
+	<application android:label="MCFPuppet">
```

Comment:
No you don't need that. You need to preserve bindings like in distribute and let SDK manifest merging do the magic.